continuous_integration/install.sh

```diff
@@ -76,7 +76,7 @@ if [ -z "$NUMPY_VERSION" ]; then
 fi
 
 if [[ "$COVERAGE" == "true" ]]; then
-    pip install coverage coveralls
+    pip install coverage coveralls memory_profiler
```

Comment:
I agree but I added it here because coveralls was failing due to my memory check functions in `test/common.py` being not covered if memory_profiler if not installed.